web/themes/custom/cloudy/src/css/_overrides.scss

```diff
@@ -6,7 +6,7 @@ Description: Bootstrap 4 with Sass
 /* TYPOGRAPHY */
 
 /* Google Fonts */
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,700,800');
```

Comment:
This needs imported into PL.

Reply:
Good catch, the font was cached on my end